src/test/config.sol

```diff
@@ -1730,4 +1730,4 @@ contract Config {
             offboarding:  false
         });
     }
-}
+}
```

Comment:
Careful with the trailing new-line removal.

Please make sure your editor has [editorconfig](https://editorconfig.org/) support to avoid that.

Reply:
updated.